test/helper.go

```diff
@@ -141,16 +141,16 @@ func NewHelperWithHooksAndDBsetup(t *testing.T, httpServer *httptest.Server, set
 		glog.Fatalf("Unable to initialize testing environment: %s", err.Error())
 	}
 
-	h.CleanDB()
```

Comment:
`CleanDB()` is [called by](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/test/helper.go#L334) `ResetDB()` (on L#145). No need to call it twice.

Reply:
:+1: